src/translations/en-US.json

```diff
@@ -856,5 +856,55 @@
         "value": "If the filter condition includes 0, blank values will be treated as 0 and included in calculations.",
         "comment": "Checkbox label tooltip. 'blank values' are data rows without any value (empty), which can be considered to be 0 during calculations. If the filter is set to '< 1' for example, it includes values of 0 (less than one) and therefore all the blank (empty) values are going to be considered to be 0 in calculations.",
         "limit": 0
+    },
+    "measureNumberFormat.inherit.optionLabel": {
+        "value": "Inherit",
+        "comment": "The label of option shown in number format presets dropdown. The label is used by an option that selects format inherited from the 'format' prop of the measure in the visualization object.",
+        "limit": 0
+    },
+    "measureNumberFormat.custom.optionLabel": {
+        "value": "Custom",
+        "comment": "The label of option shown in number format presets dropdown. The label is used by an option that opens Custom format editor window",
+        "limit": 0
+    },
+    "measureNumberFormat.buttonLabel": {
+        "value": "Format: {selectedFormatPresetName}",
+        "comment": "The label of a button that opens number format dropdown in a measure bucket item configuration. Do not translate {selectedFormatPresetName} placeholder.",
+        "limit": 0
+    },
+    "measureNumberCustomFormatDialog.title": {
+        "value": "Custom format",
+        "comment": "The title of the dialog for setting a custom measure format",
+        "limit": 0
+    },
+    "measureNumberCustomFormatDialog.template.preview.title": {
+        "value": "Format preview",
+        "comment": "The title of the help bubble for custom measure format template",
+        "limit": 0
+    },
+    "measureNumberCustomFormatDialog.template.title": {
+        "value": "Templates",
+        "comment": "The title of the list of templates for custom measure format template",
+        "limit": 0
+    },
+    "measureNumberCustomFormatDialog.extendedPreview.button": {
+        "value": "Show more",
+        "comment": "The title of button which will display more examples",
+        "limit": 0
+    },
+    "measureNumberCustomFormatDialog.howToFormat": {
+        "value": "How to format?",
+        "comment": "Link to help",
+        "limit": 0
+    },
+    "measureNumberCustomFormatDialog.definition": {
+        "value": "Definition",
+        "comment": "Title above the textarea for measure format definition",
```

Comment:
I'm afraid "above the textarea" would not help the translators who don't have the picture. Something "Header of format definition field" would be sufficient.